src/timestep.jl

```diff
@@ -1,11 +1,38 @@
+"""
+    Timestep
+
+An abstract type specifying the way a time step is chosen. 
+The following basic types are supported:
+- `FixedTimestep`: a constant, fixed number is used
+- `RandomTimestep`: the time step is random from a fixed distribution
+- `OEDPolicy`: an optimal time step is chosen based on some objective function
+"""
 abstract type Timestep end
 
+"""
+    get_step(::Timestep)
+
+Returns a value for the time step, based on the chosen method for choosing a time step.
+"""
+# NOTE: this could be simplified by making subtypes of `Timestep` callable
 function get_step(::Timestep) end
 
+"""
+    FixedTimestep(dt)
+
+Choose a fixed time step of size `dt`.
+"""
+# TODO: should we enforce positivity?
```

Comment:
@camillegontier, what do you think?

Reply:
It would indeed be nice.
Also, I realized that for the moment we can only use timesteps that are either constant, drawn from a certain distribution, or computed by OED. In practice, timesteps are usually defined as repetitive deterministic trains, eg 20 repetitions of a train of 4 spikes at 20Hz (ISI = 0.05s), followed by a recovery spike 0.5s later:

d_t = repeat([0.05,0.05,0.05,0.05,0.5],20)

Can we implement this as well ?

Reply:
Ok, will open issues for these two things.